homeassistant/components/matter/sensor.py

```diff
@@ -163,6 +163,19 @@ def _update_from_device(self) -> None:
         entity_class=MatterSensor,
         required_attributes=(clusters.PowerSource.Attributes.BatPercentRemaining,),
     ),
+    MatterDiscoverySchema(
```

Comment:
Please start with a fresh branch. This change is already merged to dev branch.

Reply:
I have since updated my branch. Is it still necessary?